ccic/processing.py

```diff
@@ -796,7 +811,7 @@ def get_encodings_zarr(variable_names):
             "_FillValue": 255,
             "dtype": "uint8",
         },
-        "cloud_type": {"compressor": compressor, "dtype": "uint8", "_FillValue": -1},
```

Comment:
This looked like a typo, `unit8` and a `-1` as a fill value should be incompatible (or automatically translated to 255)